com.unity.multiplayer.mlapi/Runtime/Core/NetworkedBehaviour.cs

```diff
@@ -22,6 +22,21 @@
 
 namespace MLAPI
 {
+    // todo: This is temporary, to be replaced by the tick system
+    static class TickSystem
+    {
+        // todo: this might belong in the tick system, in the end
+        // special value to indicate "No tick information"
+        public const ushort k_NoTick = 65535;
+        // Number of ticks over which the tick number wraps back to 0
+        public const ushort k_TickPeriod = 65000;
+
+        public static ushort GetTick()
+        {
+            return (ushort)(((long)(Time.time / 0.050)) % k_TickPeriod);
```

Comment:
Ah, would we not want the 0.050 factor to be user configurable?

Reply:
This won't stay. As soon as we get the tick system, this goes away. The tick system will definitely offer some controls, but it's not clear yet what they'll be